app/views/pages/privacy_notice.md

```diff
@@ -1,150 +1,22 @@
-
-<p class='govuk-body'>We share your anonymised personal data so that we can analyse the ITT process, its operation and any strategic or policy related studies on behalf of DfE.</p>
-
-<h2 class='govuk-heading-m' id='your-rights'>Your rights</h2>
-
-<p class='govuk-body'>Under the DPA 2018 and the UK GDPR, you have the right to find out what data we have about you. However, your rights are conditional on the lawful basis under which your data is being processed. Our lawful basis is public task, which grants you the following rights:</p>
-
-<ul class='govuk-list govuk-list--bullet'>
-  <li>being informed about how your data is being used</li>
-  <li>accessing personal data</li>
-  <li>having incorrect data corrected</li>
-  <li>restricting the processing of your data (for example where the data is inaccurate or outdated)</li>
-  <li>objecting to how your data is processed in certain circumstances, including automated processing and profiling</li>
-</ul>
-
-<p class='govuk-body'>You can find more information about how we handle personal data in our personal information charter.</p>
-
-<p class='govuk-body'>Contact the Register trainee teachers team at <a class='govuk-link' href='mailto:becomingateacher@digital.education.gov.uk'>becomingateacher@digital.education.gov.uk</a> if you have any concerns about your personal data.</p>
-
-<h2 class='govuk-heading-m' id='getting-help-raising-a-concern'>Getting help raising a concern</h2>
-
-<p class='govuk-body'>If you would like to exercise any of your rights, you can email us at <a href='mailto:becomingateacher@digital.education.gov.uk?subject=Raising a concern - personal data in Register!' class='govuk-link'>becomingateacher@digital.education.gov.uk</a></p>
-
-<p class='govuk-body'>You can also use our <a href='https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen' class='govuk-link'>contact form</a> to get in touch with our Data Protection Officer.</p>
-
-<p class='govuk-body'>If we cannot resolve your issue, you have the right to raise it with the <a href='https://ico.org.uk/' class='govuk-link'>Information Commissioner's Office (ICO)</a>.</p>
-
-<h2 class='govuk-heading-m' id='keep-up-to-date'>Keeping our privacy notice up to date</h2>
-
-<p class='govuk-body'>We’ll update this privacy notice when required. You should regularly review the notice.</p>
+<p class='govuk-body'>
+  <a class="govuk-link" target="_blank" href="https://www.gov.uk/government/publications/privacy-information-education-providers-workforce-including-teachers/5a254207-a566-44f7-ac77-6ba59fd26e04#using-your-data-to-support-teaching-as-a-career">Section 2.6 (opens in new tab)</a> tells you what we do with
```

Comment:
The link text says 'Section 2.6' but the link doesn't seem to take you to section 2.6 which feels a bit jarring.

Reply:
:+1: